src/cfnlint/cfn_json.py

```diff
@@ -119,10 +119,19 @@ def __new__(self, x, start_mark, end_mark):
             return cls.__new__(self, x)
 
         def __deepcopy__(self, memo):
-            return self
```

Comment:
Here

Reply:
The better option for this would be to make this class singular for the json and yaml parsers.  There isn't a reason to have them copied.